### PR TITLE
Suggested updates for the Basic Concepts section

### PR DESCRIPTION
**Pull request by grantnoble**
Lots of linking and line length edits. Plus some formatting and grammar.

***
<a href="https://pr-preview.s3.amazonaws.com/grantnoble/vc-data-model/pull/684.html" title="Last updated on Jul 12, 2019, 2:29 AM UTC (e19629e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/684/96dc09e...grantnoble:e19629e.html" title="Last updated on Jul 12, 2019, 2:29 AM UTC (e19629e)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1076,26 +1076,27 @@ <h2>Basic Concepts</h2>
...
         <p>
-When two software systems need to exchange data, they need to use terminology that both
-systems understand.
-As an analogy, consider how two people communicate. Both people must use the same
-language and the words they use must mean the same thing to each other. This may be
-referred to as "the context of a conversation".
+When two software systems need to exchange data, they need to use terminology
+that both systems understand. As an analogy, consider how two people
+communicate. Both people must use the same language and the words they use must
+mean the same thing to each other. This could be referred to as
```
Comment (TallTed) [inline index.html]: `could` -> `might` to avoid [RFC 2119](https://tools.ietf.org/html/rfc2119) interpretation issues

File: index.html [hunk 2]
```diff
@@ -1187,32 +1188,37 @@ <h3>Identifiers</h3>
...
-	</ul>
+the optional <code>id</code> <a>property</a> for such identifiers. The
+<code>id</code> <a>property</a> is intended to unambiguously refer to an object,
+such as a person, product, or organization. Using the <code>id</code>
+<a>property</a> allows for the expression of statements about specific things in
+the <a>verifiable credential</a>.
+        </p>
+
+        <p>
+<em>If</em> the <code>id</code> <a>property</a> is present, the:
```
Comment (TallTed) [inline index.html]: "the" belongs at the start of each bullet.  Comprehension is not increased by this split.

File: index.html [hunk 3]
```diff
@@ -1281,22 +1287,23 @@ <h3>Types</h3>
 
         <p>
 <a>Verifiable credentials</a> and <a>verifiable presentations</a> MUST have a
-<code>type</code> <a>property</a>; in other words, any <a>credential</a> or
-<a>presentation</a> that lacks a <code>type</code> <a>property</a> <em>is not
-verifiable</em>, so is not a <a>verifiable credential</a> nor a
+<code>type</code> <a>property</a>. That is, any <a>credential</a> or
+<a>presentation</a> that does not have <code>type</code> <a>property</a>
+<em>is not <a>verifiable</a></em>, so is not a <a>verifiable credential</a> or a
```
Comment (TallTed) [inline index.html]: `is not a <a>verifiable credential</a> or a`
-> `is neither a <a>verifiable credential</a> nor a`

File: index.html [hunk 4]
```diff
@@ -1667,24 +1680,30 @@ <h3>Proofs (Signatures)</h3>
...
-<a>verifiable credential</a> or <a>verifiable presentation</a>, i.e., to be
-<em>verifiable.</em> This specification
-identifies two classes of proof mechanisms: external proofs and embedded proofs. An
-<dfn>external proof</dfn> is one that wraps an expression of this data model,
-such as a JSON Web Token, which is elaborated upon in Section
-<a href="#json-web-token"></a>. An <dfn>embedded proof</dfn> is a mechanism
-where the proof is included in the data, such as a Linked Data Signature,
-which is elaborated upon in Section <a href="#linked-data-proofs"></a>.
-When embedding a proof, the following <a>property</a> MUST be used:
+<a>verifiable credential</a> or <a>verifiable presentation</a>. That is, to be
```
Comment (TallTed) [inline index.html]: `presentation</a>. That`
-> `presentation</a>; that`

File: index.html [hunk 5]
```diff
@@ -1737,7 +1756,7 @@ <h3>Proofs (Signatures)</h3>
 proof mechanisms, and this specification does not standardize nor recommend any
 single proof mechanism for use with <a>verifiable credentials</a>. Implementers
 that are interested in understanding more about the <code>proof</code>
-mechanism can learn more by reading the following specifications:
+mechanism should read the following specifications:
```
Comment (TallTed) [inline index.html]:
```
Implementers
that are interested in understanding more about the <code>proof</code>
mechanism can learn more by reading the following specifications:
mechanism should read the following specifications:
```
->
```
For more 
information about the <code>proof</code> mechanism, see the following 
specifications:
```

Again, avoiding [RFC2119](https://tools.ietf.org/html/rfc2119).

File: index.html [hunk 6]
```diff
@@ -1750,8 +1769,8 @@ <h3>Proofs (Signatures)</h3>
         <h3>Expiration</h3>
 
         <p>
-This specification defines the following optional <a>property</a> for the expression
-of <a>credential</a> expiration information:
+This specification defines the <code>expriationDate</code> <a>property</a> for
```
Comment (TallTed) [inline index.html]: `expriationDate`
-> `expirationDate`

File: index.html (hunk 1 above)
Comment (grantnoble) [reply index.html]: Done.

File: index.html (hunk 2 above)
Comment (grantnoble) [reply index.html]: Repeating text at the beginning of each bullet point, when you can have it just once in the stem sentence goes against my pedantic nature of being a Technical Writer, but done. :smile:

File: index.html (hunk 3 above)
Comment (grantnoble) [reply index.html]: Done.

File: index.html (hunk 4 above)
Comment (grantnoble) [reply index.html]: Done.

File: index.html (hunk 5 above)
Comment (grantnoble) [reply index.html]: Done.

File: index.html (hunk 6 above)
Comment (grantnoble) [reply index.html]: Good pickup! Done.

**Pull request merged**